packages/legend-graph/src/graphManager/AbstractPureGraphManager.ts

```diff
@@ -315,6 +315,9 @@ export abstract class AbstractPureGraphManager {
     lambda: RawLambda,
     runtime: Runtime,
     clientVersion: string,
+    parameters: (string | number | boolean)[],
+    // Anonymizes data by hashing any string values in the generated data
+    anonymizeGeneratedData?: boolean,
```

Comment:
as said, this is a very specific feature of the data generation endpoint, we should put it in `options`